patches/corefx/0001-wip.patch

```diff
@@ -0,0 +1,24 @@
+From d0a32957e3ebcb5462ffe783bee9af73b60373fc Mon Sep 17 00:00:00 2001
+From: Chris Rummel <crummel@microsoft.com>
+Date: Thu, 15 Nov 2018 19:27:20 -0600
+Subject: [PATCH] wip
+
+---
+ global.json | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/global.json b/global.json
+index 808d3a98ec..2c55eb7cde 100644
+--- a/global.json
++++ b/global.json
+@@ -1,6 +1,6 @@
+ {
+   "tools": {
+-    "dotnet": "2.1.401"
++    "dotnet": "2.1.403"
```

Comment:
Should UpdateCLIVersionInGlobalJson in repos/dir.targets be updated to do this rather than using a patch?

Reply:
...and now I see it's marked WIP!